share/spice/plos/plos.js

```diff
@@ -28,18 +28,34 @@
                 sourceName: "PLOS",
                 sourceUrl: 'http://www.plosone.org/search/advanced?unformattedQuery=' + query
             }, 
-	    normalize: function(item) {
-		return {
-		    url: "http://dx.doi.org/" + item.id
-		};
-	    },
+            normalize: function(item) {
+                var authors = item.author_display || [];
+                var subtitle = authors.length ? authors.join(", ") : "";
+
+                //strip html from title
+                var tmp = document.createElement('DIV');
+                tmp.innerHTML = item.title_display;
+                var title = tmp.textContent || tmp.innerText || '';
+
+                var journalName = item.journal ? item.journal + ' ' : '';
+                var issueNumber = item.issue ? 'Issue ' + item.issue : '';
```

Comment:
Can you chain multiple var declarations as suggested in the [code style guide](https://duck.co/duckduckhack/code_styleguide)